base/gmp.jl

```diff
@@ -694,6 +694,14 @@ function string(n::BigInt; base::Integer = 10, pad::Integer = 1)
     String(sv)
 end
 
+function digits(n::BigInt; base::Integer = 10, pad::Integer = 1)
+  if base ≤ 62
```

Comment:
Should this also check that `2 ≤ base`?

Reply:
Done.

Reply:
Might it work for `-62 <= base <= -2` ? (I'm asking just for curiosity, implementing fast `digits` for negative bases isn't blocking a release).